examples/attention_sink/example_gqa_sink_bwd_varlen.py

```
@@ -0,0 +1,798 @@
+    (
+        q_unpad,
+        k_unpad,
+        v_unpad,
+        cu_seqlens_q,
+        cu_seqlens_k,
+        max_seqlen_q,
+        max_seqlen_k,
+        q,
+        k,
+        v,
+        output_pad_fn,
```

Ruff (0.14.10)

Unpacked variable `output_pad_fn` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)